string[] object_names
geometry_msgs/PoseStamped[] poses
